src/main/java/com/google/sps/data/OrganizationUpdater.java

```diff
@@ -57,6 +60,7 @@ public void updateOrganization(HttpServletRequest request, GivrUser user, boolea
     properties.put("approval", "isApproved");
     properties.put("moderator-list", "moderatorList");
 
+    // Updates entity properties from form
```

Comment:
What if instead of assuming the history update in the helper, you took in an EmbeddedEntity instance here?